apps/website/src/pages/docs/radiotile.mdx

```diff
@@ -11,6 +11,8 @@ import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
 import * as AllExamples from 'examples';
 
+A series of selectable tile components that behave like [radio button](radio). You may include icons to help make each tile more identifiable.
```

Comment:
I would move this to the `description: ` section on line 3 or at least move the first sentence there

Reply:
Cut and paste

Reply:
![image](https://github.com/iTwin/iTwinUI/assets/78564221/f6a259d8-1ee8-4adc-a918-bc8cacc01e2f)

Actually I'm mistaken. It appears we can't link the `Radio` page when using this syntax in the description section. Therefore, what you had previously is fine. Sorry!

Reply:
All good :)